### DISCUSSION TITLE
Fairdrive designs

### DISCUSSION THREAD

**Discussion by crtahlin** [fairDataSociety/fairdrive-theapp#434] [General]
For reference, Fairdrive designs can be found at  [Figma](https://www.figma.com/file/1uwKXyizhv88JvDckja7Py/Fairdrive-Main?type=design&node-id=1%3A3576&mode=design&t=dgbXKFHjFvkhZ0A6-1)

**Comment by tfius** [comment]
Designs in pdf form: https://gateway.fairdatasociety.org/access/ead99032c45535e4a73a5e3dc802d438523264a0d49e5a4014a4ab96dd57ba8c

**Comment by tfius** [comment]
[Fairdrive Main Pngs.zip](https://github.com/fairDataSociety/fairdrive-theapp/files/12259094/Fairdrive.Main.Pngs.zip)
